test/rules/callable-types/test.ts.lint

```diff
@@ -1,23 +1,36 @@
 interface I {
-          ~   [Interface has only a call signature — use `type I = () => void` instead.]
     (): void;
+    ~~~~~~~~~   [Interface has only a call signature — use `type I = () => void;` instead.]
 }
 
 interface J extends Function {
-          ~                    [Interface has only a call signature — use `type J = () => void` instead.]
     (): void;
+    ~~~~~~~~~                    [Interface has only a call signature — use `type J = () => void;` instead.]
+}
+
+interface K<T> {
+    <U extends T>(param1: U, param2: T): U;
+    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ [Interface has only a call signature — use `type K<T> = <U extends T>(param1: U, param2: T) => U;` instead.]
+}
+
+interface L<T> {
+    (): T
+    ~~~~~ [Interface has only a call signature — use `type L<T> = () => T` instead.]
 }
 
 type T = {
     (): void;
-    ~~~~~~~~~ [Type literal has only a call signature — use `() => void` instead.]
+    ~~~~~~~~~ [Type literal has only a call signature — use `() => void;` instead.]
```

Comment:
it actually handles that already. The only exception is on type aliases. But since a TypeAlias should always end with a semicolon, I guess I can always remove the semicolon from the call signature